### DISCUSSION TITLE
Use of try option with gmx insert-molecules?

### DISCUSSION THREAD

**Discussion by dpadula85** [Becksteinlab/GromacsWrapper#221] [Q&A]
Hello,

I am using gmx insert-molecules through GromacsWrapper. I would need to use the `-try` option of the program, to increaase the number of attempts needed to introduce a new residue. However, I cannot do

`gromacs.insert_molecules( f=my_file1, ci=my_file2, try=100)`

because `try` is a python command.
I also tried
`gromacs.insert_molecules( f=my_file1, ci=my_file2, input=('-try 100')`

but apparently does not work.
Any suggestion on how to do it?

**Comment by orbeckst** [comment]
Hi @dpadula85 , try prefixing with underscore:
```python
gromacs.insert_molecules( f=my_file1, ci=my_file2, _try=100)
```
This hack is documented under https://gromacswrapper.readthedocs.io/en/latest/gromacs/core/core.html?highlight=underscore#gromacs.core.GromacsCommand but it's not easy to find.

**Comment by dpadula85** [comment]
Hi @orbeckst, as usual my bad for not going thoroughly through the docs. Thanks once again for your help, let me take the occasion to say that I'm a big fan of your tools!

Cheers,
D.

**Reply by orbeckst** [reply]
I converted the issue to a discussion. If the answer above helped, mark it as an answer, please.

**Comment by orbeckst** [comment]
No problem & thank you!

(The docs are not particularly good but I don't have time to improve them — any contributions are welcome ;-) ).
